### PR TITLE
[WIP] Update docs to sync with cookiecutter master branch

### PR DESCRIPTION
**Pull request by stevepiercy**
- [x] `docs/quick_tour.rst`
- [x] `docs/narr/project.rst`
- [x] `docs/quick_tutorial/*`
- [x] `docs/tutorials/wiki/*`
- [x] `docs/tutorials/wiki2/*`
- [x] Any files that refer to `docs/narr/myproject`

See #3523

### CODE REVIEW

**Comment by stevepiercy** [comment]
This monster is ready for review.

**Comment by mmerickel** [comment]
Let's say hypothetically I moved the tests out of the package. On a scale from -1 to +1 how annoyed would you be to re-sync this again?

**Comment by stevepiercy** [comment]
+1. Tests are the least troublesome to synch, as they are at the end of every tutorial. It'll have to wait till Monday night though.

**Comment by mmerickel** [comment]
OK I will try to sort it out tomorrow so that it’s ready for you on Monday. Thank you!

**Comment by stevepiercy** [comment]
If you're feeling ambitious, test coverage for both the sqlalchemy and zodb backends are low, and maybe move source into `/src/package`?

**Comment by mmerickel** [comment]
I do not want to move the source into `/src/package` - I don't do it in my own apps and I think it's mostly valuable for libraries versus apps. I have made the change to move the tests but won't do anything to increase coverage atm.

**Comment by stevepiercy** [comment]
@mmerickel I could use some help with one failing test.

https://github.com/Pylons/pyramid/blob/6c5f524cdb750a7a3a112087942a0eafc8e10e4d/docs/tutorials/wiki2/src/tests/tests/test_initdb.py#L12-L16

```
E       sqlite3.IntegrityError: UNIQUE constraint failed: users.name
```

The database already exists and has been populated with data from an earlier step in the tutorial when changing the models. If I change the database name in `development.ini` to `tutorialtest.sqlite` and adjust the test accordingly, it passes. Obviously that is not the right way, but I couldn't figure out the right way.

Also there are a couple of warnings, one for `imp`, which will go away in a future release, but there are a couple of other warnings:

```
tests/test_functional.py::FunctionalTests::test_anonymous_user_cannot_add
tests/test_functional.py::FunctionalTests::test_anonymous_user_cannot_add
  /Users/stevepiercy/projects/hack-on-pyramid/tutorial/env/lib/python3.7/site-packages/pkg_resources/__init__.py:1145: DeprecationWarning: Use of .. or absolute path in a resource path is not allowed and will raise exceptions in a future release.
    self, resource_name
```

I fiddled around with that for a while, but nothing obvious jumped out at me.

**Comment by stevepiercy** [comment]
This PR is now ready for review, and except for the above mentioned failing test and test warnings, can be merged.

**Comment by mmerickel** [comment]
The test should be either

1) removed

2) modified to just skip itself if the database already exists

in its current form, initialize_db is meant to be run only one time ever. So if the database already exists it will fail to run again. The bug here (imo) is having this test in the first place and I'd just remove it.

**Comment by stevepiercy** [comment]
Thanks. Removal was my first thought. Tests should be repeatable, and modifying the script would feel like accommodating something that doesn't need to be accommodated.

**Comment by stevepiercy** [comment]
I rebased this branch on master, resolved conflicts and merged, then force pushed. I guess that wasn't the proper procedure, looking at the commit history.

**Comment by mmerickel** [comment]
is this ready for review then or still wip?

**Comment by stevepiercy** [comment]
The only thing missing is how to resolve the warnings in https://github.com/Pylons/pyramid/pull/3556#issuecomment-570122485

I couldn't figure it out. Otherwise it is ready for review.

**Comment by merwok** [comment]
I would edit the pkg_resources code in site-packages to add/change `stacklevel` parameter to `warnings.warn` in order to find the faulty code.

**Comment by stevepiercy** [comment]
Thanks for the tip! I'll try that out tonight, after daylight runs out.

**Comment by stevepiercy** [comment]
PyCharm caching of diff will be the death of me. Refreshing cache revealed the file diff.

Thanks to the tip to modifying `pkg_resources`, I found a few files that did not show up in PyCharm's diff that used `../` instead of `tutorial:` for the `renderer`. I had to change `msg = "Use of .. or absolute path in a resource path is not allowed."` to include `str(path.split(posixpath.sep))`, which revealed the offending files.

Changes pushed, and this PR is now complete and ready for review. Ping @mmerickel.

File: docs/narr/project.rst [hunk 1]
```diff
@@ -1159,11 +1166,9 @@ inotify support
...
 
 By default ``hupper`` will poll the filesystem for changes to all Python
-code. This can be pretty inefficient in larger projects. To be nicer to your
-hard drive, you should install the
-`watchdog <https://pythonhosted.org/watchdog/>`_ package in development.
-``hupper`` will automatically use ``watchdog`` to more efficiently poll the
-filesystem.
+code. This can be pretty inefficient in larger projects.
+To be nicer to your hard drive, you should install either the `watchman <https://facebook.github.io/watchman/>`_ or `watchdog <https://pythonhosted.org/watchdog/>`_ package in development.
+``hupper`` will use, in order of preference for efficiency, if available, ``watchman``, ``watchdog``, then finally ``inotify`` to poll the filesystem.
```
Comment (mmerickel) [inline docs/narr/project.rst]: watchman, watchdog or simple polling

**Comment by mmerickel** [comment]
I pushed one small tweak to find_packages. This is great work. I'm going to merge it and will try to update the tutorials this weekend.

**Pull request merged**